docs/architecture/README-copy.md

```diff
@@ -0,0 +1,303 @@
+# Overview
+
+This document describes the high-level architecture of nearcore. The focus here
+is on the implementation of the blockchain protocol, not the protocol itself.
+For reference documentation of the protocol, please refer to
+[nomicon](https://nomicon.io/)
+
+Some parts of our architecture are also covered in this [video series on YouTube](https://www.youtube.com/playlist?list=PL9tzQn_TEuFV4qlts0tVgndnytFs4QSYo).
+
+## Bird's Eye View
+
+If we put the entirety of nearcore onto one picture, we get something like this:
+
+![](../images/architecture.svg)
```

Comment:
The drawback of this approach is that images are not being rendered as above.  I am not aware of a github where we can view all inline comments and have the images rendered at the same time.  I think this really is main feature that sets google docs apart from github discussions.